### ISSUE TITLE
[Solved] 18.2.8 - 19.1.X - Text boxes and number boxes at low resolutions (almost) entirely hides text

### ISSUE THREAD

**Issue by briosheje** [DevExpress/DevExtreme#8246]
**TL;DR** Be sure that the css you're using is not generated with the **old** devextreme theme builder, otherwise components may act in a weird way.

### Steps to Reproduce:
1. Use any dx-text-box or number box with the clear button enabled.
2. Go to low resolutions like 1024x768
3. Use bootstrap and try to work with md-4 or md-3 or whatever.

### Results You Received:
The text is almost entirely truncated (below screens targets 18.2.8).

Number:
![image](https://user-images.githubusercontent.com/1898880/58687045-c83cef80-8380-11e9-8c3f-81f57ed8ceaa.png)

Bigger number box:
![image](https://user-images.githubusercontent.com/1898880/58687094-f4587080-8380-11e9-861b-e0a28338f66e.png)

Number box with spinners:
![image](https://user-images.githubusercontent.com/1898880/58687257-6cbf3180-8381-11e9-8c02-6b81303fa2a8.png)

TextBox inside DropDownBox:
![image](https://user-images.githubusercontent.com/1898880/58687107-ffab9c00-8380-11e9-9651-f7f98cb89367.png)

This is quite crucial, since most of the number boxes, at low resolutions, should be quite small in order to don't take unnecessary space. Previous versions of devextreme didn't suffer such an issue.

### Results You Expected:
I would not expect the text to be truncated by the buttons overlay.

### Environment Details:
Browsers tested:
- Chrome
- Firefox

**Comment by briosheje** [comment]
Update:

The issue seems caused by a strange padding of `.dx-texteditor-input`:

![image](https://user-images.githubusercontent.com/1898880/58688771-aa25be00-8385-11e9-8222-c20474fbf8fd.png)

This looks weird, it seems the padding is calculated **before** the floating buttons, so there is an extra padding happening.

As a side note, I'm not using the dxButton `buttons` property, which seems to fix the issue in the official example given here: https://js.devexpress.com/Demos/WidgetsGallery/Demo/Common/CustomTextEditorButtons/jQuery/Light/ where the padding seems to be calculated properly.

**Comment by briosheje** [comment]
Update: even by setting `buttons: ['clear']` only, the padding is still 66, I can't really figure out what's happening, but I can't really foresee any cause.

![image](https://user-images.githubusercontent.com/1898880/58689248-c6762a80-8386-11e9-8e9c-22c403f903a0.png)


The cause seems to be this:

![image](https://user-images.githubusercontent.com/1898880/58689817-57013a80-8388-11e9-9f43-a2e9c336fe0d.png)

I'm not exactly sure where it's coming from, but it's giving a huge padding from the right.

**Comment by briosheje** [comment]
Okay, I've found the issue: I'm using a custom CSS theme generated through the Theme Generator.

**however**, the theme was generated using an older version, since I'm using such theme since verison 18.1.7.

![image](https://user-images.githubusercontent.com/1898880/58689998-d55ddc80-8388-11e9-80fa-6a179e8f3034.png)

This css line is causing the issue. I will generate the theme again, I didn't think the theme generation would impact the component behaviors, hence the problem.

**Comment by briosheje** [comment]
Ok, I can confirm that updating the scss theme using the new ThemeBuilder fixes the issue:

![image](https://user-images.githubusercontent.com/1898880/58690293-81072c80-8389-11e9-9c93-0e0e435bab02.png)

**Comment by briosheje** [comment]
@DokaRus Perhaps, in the docs, you may want to mention that custom SCSS or whatever SCSS generated from the tool (either the old theme generator, either the new one) should be regenerated each major / minor release. In that case, the issue appeared in a minor (or patch, i guess) release, hence it was not that obvious to guess that the issue came from the scss.
